apps/site-landing/internals/server.mjs

```diff
@@ -4,9 +4,10 @@ import { join, dirname } from "path";
 import { readFileSync } from "fs";
 import { fileURLToPath } from "url";
 
-const __filename = fileURLToPath(import.meta.url);
-const __dirname = dirname(__filename);
-const datapath = join(__dirname, '..', 'src', 'containers', 'BenefitsSimulator', 'simulator');
+const url = import.meta.url;
```

Comment:
`unexpected reserved word` when passing import.meta.url directly to fileUrlToPath